nautobot/core/nautobot_app/__init__.py

```diff
@@ -0,0 +1,169 @@
+from django.apps import AppConfig
+from django.urls import reverse
+from operator import getitem
+from collections import OrderedDict
+
+from nautobot.extras.choices import CustomLinkButtonClassChoices
+from nautobot.extras.plugins.utils import import_object
+from nautobot.extras.registry import registry
+
+
+class NautobotConfig(AppConfig):
+    menu_tabs = "navigation.menu_tabs"
+
+    def ready(self):
+        if self.__module__.endswith(".apps"):
+            self.__module__ = self.__module__[:-5]
```

Comment:
What is going on here. Any time I see indexing and dunkers together I get frightened! Can you please add a comment here explaining what this does and why it is necessary?

Reply:
Second Jathan's comments. We shouldn't be modifying `self.__module__`, it should be considered as read-only. This feels like something's wrong with our initialization pattern, tbh.